modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase

```diff
@@ -104,13 +106,48 @@ definition {
 		// TODO LPS-123539 CheckTranslationOfFieldsetOnWebContent pending implementation
 	}
 
-	@description = "This is a test for LPS-123539. As Content Creator, I want to leverage the structures’ translations with fildset when creating a new Web Content"
-	@ignore = "true"
+	@description = "This is a test for LPS-123539. As Content Creator, I want to leverage the structures’ translations with fieldset when creating a new Web Content"
 	@priority = "4"
 	test DefaultTitleIsDisplayedUponLocaleChange {
-		property portal.acceptance = "false";
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(
+			structureDescription = "Structure Description Test",
+			structureName = "Structure FieldSet Test");
+
+		DataEngine.addField(
+			fieldFieldLabel = "Text Test",
+			fieldName = "Text");
+
+		WebContentStructures.saveCP(structureName = "Structure Name Test");
+
+		WebContentStructures.addCP(
+			structureDescription = "Structure Description Test",
+			structureName = "Structure Name Test");
+
+		DataEngine.addFieldsets(
+			fieldsetLabel = "Structure FieldSet Test",
+			fieldsetName = "Structure FieldSet Test");
+
+		WebContentStructures.saveCP(structureName = "Structure Name Test");
+
+		NavItem.gotoWebContent();
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "Structure Name Test");
 
-		// TODO LPS-123539 DefaultTitleIsDisplayedUponLocaleChange pending implementation
+		PortletEntry.inputTitle(title = "WC Title Test");
+
+		PortletEntry.changeLocale(locale = "pt_BR");
+
+		AssertTextEquals(
+			locator1 = "TextInput#TITLE",
+			value1 = "WC Title Test");
+
+		PortletEntry.changeLocale(locale = "es_ES");
+
+		AssertTextEquals(
+			locator1 = "TextInput#TITLE",
+			value1 = "WC Title Test");
 	}
```

Comment:
Add step to publish web content then check default title is shown in default language and non translated language